include/ck_tile/ops/layernorm2d/pipeline/layernorm2d_fwd_pipeline_one_pass.hpp

```diff
@@ -110,6 +117,12 @@ struct Layernorm2dFwdPipelineOnePass
                      kFusedAdd == Layernorm2dFusedAddEnum::PRE_ADD)
         {
             sweep_tile(x_resi, [&](auto idx) {
+                if constexpr(kBias == Layernorm2dBiasEnum::ADD_BIAS)
```

Comment:
`Layernorm2dBiasEnum::ADD_BIAS` is under the condition of `kFusedAdd` is either `PRE_ADD_STORE` or `PRE_ADD` (line:116~117),  it means the bias actually has dependency of pre-add(store), or fused shortcut. I think these features are independent?